<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adjust</class>
 <widget class="QWidget" name="Adjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="X_label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="X_spinBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="X_horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Y_label">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="Y_spinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="Y_horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Z_label">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="Z_spinBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QSlider" name="Z_horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
